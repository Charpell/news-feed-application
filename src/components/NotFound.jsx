import React from 'react';

/**
 * return a not found page
 * @returns {void}
 */
function NotFound() {
  return (
    <div className="not-found">
      <div className="frown">
        <i className="fa fa-frown-o" aria-hidden="true" />
      </div>
      <div className="four">404</div>
      <div className="four-text">
        <h1>Oops, we couldn't find that page!</h1>
        <p>It looks like nothing was found at this Location</p>
      </div>
    </div>
  );
}

export default NotFound;
