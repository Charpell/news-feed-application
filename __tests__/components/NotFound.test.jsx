import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../../src/components/NotFound';

describe('NotFound Page', () => {
  it('renders a snapshot', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
