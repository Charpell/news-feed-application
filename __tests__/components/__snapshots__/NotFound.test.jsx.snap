// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound should render correctly 1`] = `
<div
  className="not-found"
>
  <div
    className="frown"
  >
    <i
      aria-hidden="true"
      className="fa fa-frown-o"
    />
  </div>
  <div
    className="four"
  >
    404
  </div>
  <div
    className="four-text"
  >
    <h1>
      Oops, we couldn't find that page!
    </h1>
    <p>
      It looks like nothing was found at this Location
    </p>
  </div>
</div>
`;
